src/openrct2/park/ParkFile.h

```diff
@@ -9,10 +9,10 @@ struct ObjectRepositoryItem;
 namespace OpenRCT2
 {
     // Current version that is saved.
-    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 22;
+    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 23;
```

Comment:
unfortunately we just had to bump it to 23 could you rebase one more time and make it 24.